src/patterns/components/slideshow/base.hbs

```diff
@@ -0,0 +1,45 @@
+<div class="u-bgGray u-posRelative Slideshow js-Slideshow u-pullSides1">
+  <ul class="u-flex u-listUnstyled Slideshow-slides js-Slideshow-slides u-padEnds2">
+    {{#each drizzle.data.fun-facts.contents.aileen}}  
+      <li class="Grid Grid--withGutter Grid--alignMiddle Slideshow-slide js-Slideshow-slide u-flexExpand u-flexJustifyCenter u-sm-flexJustifyStart u-padSides5 {{#if @first}}is-visible{{/if}}">
+        <div class="Grid-cell u-size1of2 u-sm-size1of4 u-lg-size1of5">
+          <div class="Thumbnail Thumbnail--circle u-block">
+            <img class="u-sizeFull" src="{{image}}" alt="">
+          </div>
+        </div>
+        <div class="Grid-cell u-sm-size3of4 u-lg-size1of2 u-spaceItems06 u-textCenter u-sm-textLeft u-spaceTop01 u-sm-spaceTopNone">
+          <h4 class="u-md-textGrow1">Did you know&hellip;?</h4>
+          <p>{{fact}}</p>
+        </div>
+      </li>
+    {{/each}}
+  </ul>
+
```

Comment:
Note that this is totally not accessible as it sits. That's next up on the TODO list.